pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -19,6 +19,10 @@ import (
 
 const (
 	host               = "localhost"
+	downloadHost       = "https://downloads-openshift-console.apps.some.cluster.openshift.com"
```

Comment:
To be honest, for the test I'd rather inline these.  Its a tad repetitious, but I think it makes a better test if the `input` and `output` are specific to each test and the `output` (ie, the want) is hard-coded in the specific test.

Reply:
It may be good to add one test w/o the downloads route (`nil`) and make sure the config is still properly generated:

```golang
{
     name: "Test default configmap, no customization overrides",
     // downloadsRoute: &routev1.Route{ ... }
}, {
     name: "Test default configmap, without downloads",
     // downloadsRoute: nil
}
```

Reply:
> It may be good to add one test w/o the downloads route (nil) and make sure the config is still properly generated:

All other TCs have the download route set :)

Reply:
Also the we have the `host` const here